### ISSUE TITLE
Suggestion for Repair Skill

### ISSUE THREAD

**Issue by Theundead565** [mcMMO-Dev/mcMMO#1344]
Ok, so I've been using McMMo and playing on McMMo servers for quite a while, and have got to say, repair is one of the greatest skills to use, but here's an idea that I believe to be a good one.

Basic Idea:
When you have a sword, or a pickaxe, or whatever you have that is enchanted, and you want the enchantments back, you could use said item on say, a Lapis Lazuli block, and it would take all the enchantments off of the item and give you the item + the enchantment books for said item. Of course this would have to follow same thing that Salvaging does (which is that the item has to be at full durability!) or this could be abused way to easily.

Why it should be implemented:
I think if you guys could implement it, it would be great because say you get a bad enchantment on that of a Pickaxe (say Silk Touch I and Unbreaking III) and you want unbreaking III on a different pickaxe and Silk Touch on a shovel. This would be a great way to do that with Repair.

More In-Depth:
What i was thinking for this adaption of Repair, is that each item (Stone, Iron, Gold, Chain, Diamond) have different tiers, such as in fishing, that say 200 repair can do this for stone tools, 400 for iron and chain, 600 for gold (because gold is ridiculously enchant-able, so it should be high up on the tier system) and 800 for diamond. You could possibly be adding the same thing you did for Arcane Forging onto this which would be that there's a chance that the enchantment that your salvaging is "downgraded" in which Unbreaking III would be lost, but you would get Unbreaking III, with the probability of keeping the enchantments on the book the same higher on each rank.

TL;DR 
Lapis Lazuli blocks should beable to be used like a golden block can be used currently, but used to get enchantments back
